android/app/src/main/java/com/reactnativeapp/MainActivity.java

```diff
@@ -1,6 +1,19 @@
 package com.reactnativeapp;
 
+import android.os.Bundle;
+
+import com.facebook.FacebookSdk;
 import com.facebook.react.ReactActivity;
+import com.trinerdis.skypicker.logging.configuration.Configuration;
+import com.trinerdis.skypicker.logging.event.model.AppConfig;
+import com.trinerdis.skypicker.logging.sender.AppsFlyerEventSender;
+import com.trinerdis.skypicker.logging.sender.CommonExponeaEventSender;
+import com.trinerdis.skypicker.logging.sender.EventSender;
+import com.trinerdis.skypicker.logging.sender.FacebookEventSender;
+import com.trinerdis.skypicker.logging.sender.LogmoleEventSender;
+import com.trinerdis.skypicker.logging.sender.MobileExponeaEventSender;
```

Comment:
So are you Trinerdis, or Skypicker/Kiwi now?

Reply:
The whole kiwi.com Android app uses that package for historical reasons.